graphql/src/test/kotlin/com/trib3/graphql/execution/LeakyCauldronHooksTest.kt

```diff
@@ -177,22 +171,16 @@ class LeakyCauldronHooksTest {
     fun testLocalDateTime() {
         val result = graphQL.execute("""query {localDateTime(l:"2019-10-30T00:01")}""")
             .getData<Map<String, String>>()
-        assertThat(result["localDateTime"]).isEqualTo("2019-10-31T01:01")
-        assertThat {
-            graphQL.execute("""query {localDateTime(l:123)}""")
-        }.isFailure().isInstanceOf(CoercingSerializeException::class)
-
-        assertThat {
-            graphQL.execute("""query {localDateTime(l:"123")}""")
-        }.isFailure().isInstanceOf(CoercingSerializeException::class)
+        assertThat(result["localDateTime"]).isEqualTo("2019-10-31T01:01:00")
+        assertValidationErrors("""query {localDateTime(l:123)}""", """query {localDateTime(l:"123")}""")
 
         assertThat {
             LOCAL_DATETIME_SCALAR.coercing.serialize(123)
         }.isFailure().isInstanceOf(CoercingSerializeException::class)
 
         assertThat {
             LOCAL_DATETIME_SCALAR.coercing.serialize(LocalDateTime.of(2019, 10, 31, 1, 1))
-        }.isSuccess().isEqualTo("2019-10-31T01:01")
+        }.isSuccess().isEqualTo("2019-10-31T01:01:00")
```

Comment:
interested in seeing if this breaks anything.

Reply:
it doesn't in testing our downstream apps, and feels like it shouldn't too, but bumping the version for safety's sake should help protect anything it might break